lightkurve/targetpixelfile.py

```diff
@@ -407,7 +462,8 @@ def centroids(self, aperture_mask='pipeline'):
         return col_centr, row_centr
 
     def plot(self, ax=None, frame=0, cadenceno=None, bkg=False, aperture_mask=None,
-             show_colorbar=True, mask_color='pink', style='fast', **kwargs):
+             threshold=3, show_colorbar=True, mask_color='pink', style='fast',
```

Comment:
Why is threshold now needed in the plot function? Shouldn't we be passing the aperture mask instead? 

Let's have a keyword `aperture_mask` with a default of `tpf.pipeline_aperture`